azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -33,9 +33,9 @@
 public final class CallCompositeLocalOptions {
     private CallCompositeParticipantViewData participantViewData = null;
     private CallCompositeSetupScreenViewData setupScreenViewData = null;
-    private Boolean cameraOnByDefault = false;
-    private Boolean microphoneOnByDefault = false;
-    private Boolean bypassSetupScreen = false;
+    private boolean cameraOn = false;
```

Comment:
We should use private Boolean cameraOnByDefault = null;
If Contoso do not provide value, then we apply default behaviour.